packages/firebase_remote_config/lib/src/remote_config.dart

```diff
@@ -0,0 +1,235 @@
+part of firebase_remote_config;
+
+/// The entry point for accessing Remote Config.
+///
+/// You can get an instance by calling [RemoteConfig.instance]. Note
+/// [RemoteConfig.instance] is async.
+class RemoteConfig extends ChangeNotifier {
+  @visibleForTesting
+  static const MethodChannel channel =
+      const MethodChannel('plugins.flutter.io/firebase_remote_config');
+
+  static const String defaultValueForString = '';
+  static const int defaultValueForInt = 0;
+  static const double defaultValueForDouble = 0.0;
+  static const bool defaultValueForBool = false;
+
+  static RemoteConfig _instance;
```

Comment:
You can make this a local variable in `_getRemoteConfigInstance()` now that you have `_instanceCompleter`